compiler/rustc_feature/src/active.rs

```diff
@@ -688,6 +688,9 @@ declare_features! (
     /// not changed from prior instances of the same struct (RFC #2528)
     (incomplete, type_changing_struct_update, "1.58.0", Some(86555), None),
 
+    /// Tells rustdoc to automatically generate `#[doc(cfg(...))]`.
+    (active, doc_auto_cfg, "1.58.0", Some(43781), None),
```

Comment:
Since this was `beta-accepted`, I took the liberty of adjusting it to "1.57.0" in #90938. Would someone care to fix that on master as well?